rita/src/rita_common/port_emissary/mod.rs

```diff
@@ -3,22 +3,35 @@ use std::io::prelude::*;
 use std::u16;
 
 /// Helper function for parsing out port number from local_address column
-fn take_local_port(s: &str) -> u16 {
+fn take_local_port(s: &str) -> Result<u16, &'static str> {
```

Comment:
if you look in kernel interface you'll see our error enum that we use to wrap things like this, also this file should be moved over to kernel interface.